hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md

```diff
@@ -1576,6 +1576,81 @@ Why explicitly declare a bucket bound to the central endpoint? It ensures
 that if the default endpoint is changed to a new region, data store in
 US-east is still reachable.
 
+## <a name="accesspoints"></a>Configuring S3 AccessPoints usage with S3a
+S3a now supports [S3 Access Point](https://aws.amazon.com/s3/features/access-points/) usage which
+improves VPC integration with S3 and simplifies your data's permission model because different
+policies can be applied now on the Access Point level. For more information about why to use them
+make sure to read the official documentation.
+
+Accessing data through an access point, is done by using its ARN, as opposed to just the bucket name.
+You can set the Access Point ARN property using the following configuration property:
+```xml
+<property>
+    <name>fs.s3a.accesspoint.arn</name>
+    <value> {ACCESSPOINT_ARN_HERE} </value>
+    <description>Configure S3a traffic to use this AccessPoint</description>
+</property>
+```
+
+Be mindful that this configures **all access** to S3a, and in turn S3, to go through that ARN.
+So for example `s3a://yourbucket/key` will now use your configured ARN when getting data from S3
+instead of your bucket. The flip side to this is that if you're working with multiple buckets
```

Comment:
presumably you'd always want to do a per-bucket mapping, as otherwise you don't get 1 AP ARN.

Maybe we should mandate that it MUST always be on a per-bucket basis (looked up via a conf.getTrimmedString("fs.s3a." + bucket + ".access-point.arn", "")

we would have no global ARN, so no bugreps about always getting the same bucket

Reply:
I like it. Will update PR